### ISSUE TITLE
[🐛 Bug]: BrowserStack false error about `ACCESSIBILITY Build creation failed null`

### ISSUE THREAD

**Issue by dprevost-LMI** [webdriverio/webdriverio#14525]
### Have you read the Contributing Guidelines on issues?

- [x] I have read the [Contributing Guidelines on issues](https://github.com/webdriverio/webdriverio/blob/main/CONTRIBUTING.md#reporting-new-issues).

### WebdriverIO Version

9.15.0

### Node.js Version

N/A

### Mode

Standalone Mode

### Which capabilities are you using?

```typescript
{
    platformName: 'android',
    'appium:deviceName': '^(?!REDMI|Samsung Galaxy Tab|OnePlus 12R).*$',
    'appium:platformVersion': '1[345]',
    'appium:autoGrantPermissions': true,
    'wdio:specs': [ '../../src/tests/all/**/*.test.ts' ],
    'goto:caps': {
      isLocalRun: false,
      iosEnterpriseBuild: false,
      testUserEmail: '',
      isDevMode: true
    },
    'bstack:options': {
      projectName: 'REDACTED',
      buildName: 'REDACTED',
      sessionName: 'BStack wdio tests',
      debug: true,
      networkLogs: false,
      interactiveDebugging: true,
      appiumLogs: false,
      maskCommands: 'setValues, getValues, setCookies, getCookies'
    },
    'appium:settings[allowInvisibleElements]': true,
    'appium:locale': 'US',
    'appium:language': 'en'
  }
```

### What happened?

Browser stack report error `ACCESSIBILITY Build creation failed null` while this has 0 impact on our tests, and we do not care about accessibility.

According to the logs, our product map has `accessibility: null`, which suggests that the product is not enabled or not working for us.

[This PR](https://github.com/webdriverio/webdriverio/pull/14438) auto-enabled accessibility features, which, with the above, I doubt is the proper approach. Either the working conditions should be reinforced, or a simple disable flag should be implemented, so that when not needed, it can be disabled.

### What is your expected behavior?

Not see a browser stack error for the service ACCESSIBILITY I do not need.

### How to reproduce the bug.

1. Upgrade to 9.15.0
2. Launch a test from my machine to BrowserStack
3. Observe the log 
4. The error is output for `@wdio/browserstack-service`

### Relevant log output

```typescript
...
processLaunchBuildResponse called with response: {
  observability: {
    success: true,
    options: { allow_screenshots: true },
    code: null,
    errors: null,
    error_trace_id: null
  },
  accessibility: null,
  automate: null,
  build_hashed_id: ‘REDACTED’,
  jwt: 'REDACTED',
  discard_build: null,
  app_automate: {
    success: true,
    options: { allow_screenshots: true },
    code: null,
    errors: null,
    error_trace_id: null
  }
}
...
2025-06-01T15:03:27.984Z ERROR @wdio/browserstack-service: ACCESSIBILITY Build creation failed null
...
2025-06-01T15:03:27.390Z DEBUG @wdio/browserstack-service: Sending SDK event with data {
  userName: '[REDACTED]',
  accessKey: '[REDACTED]',
  event_type: 'SDKTestAttempted',
  detectedFramework: 'WebdriverIO-mocha',
  event_properties: {
    sdkRunId: ‘REDACTED’,
    testhub_uuid: undefined,
    language_framework: 'WebdriverIO_mocha',
    referrer: 'WebdriverIO-mocha/9.14.0',
    language: 'WebdriverIO',
    languageVersion: 'v18.20.4',
    buildName: 'REDACTED’,
    buildIdentifier: 'undefined',
    os: 'Darwin',
    hostname: 'REDACTED’',
    productMap: {
      observability: true,
      accessibility: null,
      percy: false,
      automate: false,
      app_automate: true
    },
    product: [ 'observability', 'app-automate' ],
    framework: 'mocha'
  }
}
```

### Code of Conduct

- [x] I agree to follow this project's Code of Conduct

### Is there an existing issue for this?

- [x] I have searched the existing issues

**Comment by christian-bromann** [comment]
cc @webdriverio/browserstack

**Comment by kamal-kaur04** [comment]
@dprevost-LMI We have initiated fix at our end, @christian-bromann must be releasing them soon.
v8: [fix for accessibility build error [wdio-browsertstack-service] [v8] by rounak610 · Pull Request #14558 · webdriverio/webdriverio](https://github.com/webdriverio/webdriverio/pull/14558) 
v9: [fix for accessibility build error [wdio-browsertstack-service] by rounak610 · Pull Request #14557 · webdriverio/webdriverio](https://github.com/webdriverio/webdriverio/pull/14557)

**Comment by christian-bromann** [comment]
Released in v8.45.0

**Comment by dprevost-LMI** [comment]
Just upgrade to the latest, and it is working well now. Thanks!
